github/installation.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
```

Comment:
On second thought, does this need its own separate file?
How about defining the `Installation` struct with the `IntegrationEvent` in `event_types.go`?